jdi-light-mobile/src/main/java/com/epam/jdi/mobile/driver/get/DriverData.java

```diff
@@ -44,92 +53,84 @@
  */
 public class DriverData {
     public static final String PROJECT_PATH = path("");
-    public static String SRC_PATH = mergePath(PROJECT_PATH,"src", "main");
-    public static String TEST_PATH = mergePath(PROJECT_PATH, "src" ,"test");
+    public static String SRC_PATH = mergePath(PROJECT_PATH, "src", "main");
+    public static String TEST_PATH = mergePath(PROJECT_PATH, "src", "test");
     public static String LOGS_PATH = mergePath(TEST_PATH, ".logs");
     public static String DRIVERS_FOLDER;
+
     public static String getDriverFolder() {
         return isNotBlank(DRIVERS_FOLDER) && !DRIVERS_FOLDER.equalsIgnoreCase("default")
-                ? DRIVERS_FOLDER : mergePath(TEST_PATH,"resources", "drivers");
+                ? DRIVERS_FOLDER : mergePath(TEST_PATH, "resources", "drivers");
     }
+
     public static String DOWNLOADS_DIR = mergePath(TEST_PATH, "resources", "downloads");
     public static PageLoadStrategy PAGE_LOAD_STRATEGY = NORMAL;
     public static String BROWSER_SIZE = "MAXIMIZE";
     public static final String DEFAULT_DRIVER = "chrome";
     public static String DRIVER_NAME = DEFAULT_DRIVER;
     public static String ARGUMENTS_PROPERTY = "arguments";
 
-    public static Map<String,String> CAPABILITIES_FOR_IE = new HashMap<>();
-    public static Map<String,String> CAPABILITIES_FOR_CHROME = new HashMap<>();
-    public static Map<String,String> CAPABILITIES_FOR_FF = new HashMap<>();
-    public static Map<String,String> CAPABILITIES_FOR_EDGE = new HashMap<>();
-    public static Map<String,String> CAPABILITIES_FOR_OPERA = new HashMap<>();
-    public static Map<String,String> COMMON_CAPABILITIES = new HashMap<>();
+    public static Map<String, String> CAPABILITIES_FOR_IE = new HashMap<>();
+    public static Map<String, String> CAPABILITIES_FOR_CHROME = new HashMap<>();
+    public static Map<String, String> CAPABILITIES_FOR_FF = new HashMap<>();
+    public static Map<String, String> CAPABILITIES_FOR_EDGE = new HashMap<>();
+    public static Map<String, String> CAPABILITIES_FOR_OPERA = new HashMap<>();
+    public static Map<String, String> CAPABILITIES_FOR_MOBILE = new HashMap<>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4973196)